.gitignore

```diff
@@ -20,3 +20,6 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+.idea*
```

Comment:
you may want to copy .gitingore from previous repo, which contains more, eg. eclipse files .,